include/worker.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2017 Drew Walters <drewwalters96@gmail.com>
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef WORKER_H
+#define WORKER_H
+
+/* Queue worker function capacity. */
+#define MAX_WORKERS 1
+
+/*
```

Comment:
Please use javadoc-style block comments (opening with `/**`) (and I need to document this). See www.stack.nl/~dimitri/doxygen/manual/docblocks.html.